src/Internal.AspNetCore.Sdk/build/Common.targets

```diff
@@ -56,4 +57,16 @@ For single-tfm projects, this will be imported from build/Internal.AspNetCore.Sd
     </Sdk_GenerateResxDesignerFiles>
   </Target>
 
+  <Target Name="_AppendCommitHashToDescription" BeforeTargets="GetAssemblyAttributes" DependsOnTargets="ResolveCommitHash">
+    <PropertyGroup>
+      <Description>$(Description) Commit hash: $(CommitHash)</Description>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="_AppendCommitHashToPackageDescription" DependsOnTargets="ResolveCommitHash">
+    <PropertyGroup>
+      <PackageDescription>$(PackageDescription) Commit hash: $(CommitHash)</PackageDescription>
```

Comment:
I'm not sure this commit hash is all that useful in this form. Commit hash of what? I think it would be better to give a link to the github page that has this commit. `$(RepositoryUrl)/tree/$(RepositoryCommit)`

Also, add a newline between package description and this detail